Failures in protocol tests

I get those failures on Python 2.7:
```
______________________________________________________________________ test_compress_numpy _______________________________________________________________________
Traceback (most recent call last):
  File "/home/antoine/distributed/distributed/protocol/tests/test_numpy.py", line 153, in test_compress_numpy
    assert all(c == 'blosc' for c in
AssertionError: assert False
 +  where False = all(<generator object <genexpr> at 0x7f1c97d24e60>)
__________________________________________________________ test_compression_takes_advantage_of_itemsize __________________________________________________________
Traceback (most recent call last):
  File "/home/antoine/distributed/distributed/protocol/tests/test_numpy.py", line 204, in test_compression_takes_advantage_of_itemsize
    assert sum(map(len, aa)) < sum(map(len, bb))
AssertionError: assert 4001396 < 4001396
 +  where 4001396 = sum([4001396])
 +    where [4001396] = map(len, ["\x00\x12z\x00\x13\x00\x01\x00\x13\x01\x08\x00\x13\x02\x08\x00\x13\x03\x08\x00\x13\x04\x08\x00\x13\x05\x08\x00\x13\x0...8\x08\x00\x139\x08\x00\x13:\x08\x00\x13;\x08\x00\x13<\x08\x00\x13=\x08\x00\x13>\x08\x00\x80?B\x0f\x00\x00\x00\x00\x00"])
 +  and   4001396 = sum([4001396])
 +    where [4001396] = map(len, ["\x00\x12z\x00\x13\x00\x01\x00\x13\x01\x08\x00\x13\x02\x08\x00\x13\x03\x08\x00\x13\x04\x08\x00\x13\x05\x08\x00\x13\x0...8\x08\x00\x139\x08\x00\x13:\x08\x00\x13;\x08\x00\x13<\x08\x00\x13=\x08\x00\x13>\x08\x00\x80?B\x0f\x00\x00\x00\x00\x00"])
________________________________________________________________ test_maybe_compress_memoryviews _________________________________________________________________
Traceback (most recent call last):
  File "/home/antoine/distributed/distributed/protocol/tests/test_protocol.py", line 227, in test_maybe_compress_memoryviews
    assert compression == 'blosc'
AssertionError: assert 'lz4' == 'blosc'
  - lz4
  + blosc
```

Yet I have both lz4 and blosc installed:
```
$ python -c "import blosc, lz4"
$ 
```

--- a/distributed/protocol/tests/test_numpy.py
+++ b/distributed/protocol/tests/test_numpy.py
@@ -138,6 +138,8 @@ def test_itemsize(dt, size):
     assert itemsize(np.dtype(dt)) == size
 
 
+@pytest.mark.skipif(sys.version_info[0] < 3,
+                    reason='numpy doesnt use memoryviews')
 def test_compress_numpy():
     pytest.importorskip('lz4')
     x = np.ones(10000000, dtype='i4')
@@ -189,6 +191,8 @@ def test_dumps_large_blosc(c, s, a, b):
     result = yield x
 
 
+@pytest.mark.skipif(sys.version_info[0] < 3,
+                    reason='numpy doesnt use memoryviews')
 def test_compression_takes_advantage_of_itemsize():
     blosc = pytest.importorskip('blosc')
     x = np.arange(1000000, dtype='i8')

